### PR TITLE
DBZ-1595 Use dots as separator in param name

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1595

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java [hunk 1]
```diff
@@ -904,7 +915,7 @@ protected String slotName() {
     }
 
     protected boolean dropSlotOnStop() {
-        return getConfig().getBoolean(DROP_SLOT_ON_STOP);
+        return getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name()) ? getConfig().getBoolean(DROP_SLOT_ON_STOP_OBSOLETE) : getConfig().getBoolean(DROP_SLOT_ON_STOP);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: If both are given, old and new option, the latter should take precedence.

Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: @gunnarmorling Good point! Added

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java [hunk 2]
```diff
@@ -904,6 +915,12 @@ protected String slotName() {
     }
 
     protected boolean dropSlotOnStop() {
+        if (getConfig().hasKey(DROP_SLOT_ON_STOP.name())) {
+            return getConfig().getBoolean(DROP_SLOT_ON_STOP);
+        }
+        if (getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name())) {
+            return getConfig().hasKey(DROP_SLOT_ON_STOP_OBSOLETE.name());
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: hasKey() -> getBoolean()

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java [hunk 3]
```diff
@@ -904,6 +915,12 @@ protected String slotName() {
...
+        }
         return getConfig().getBoolean(DROP_SLOT_ON_STOP);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: Hum, this seems redundant with the first one above?

Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: Ah, is it to apply the default? If so, a comment would be good, it looks a bit confusing.

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java (hunk 2 above)
Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: @ Seems that copy and paste is more prone to bugs :-(

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java (hunk 3 above)
Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java]: Comment added

**Comment by gunnarmorling** [comment]
Rebased and applied. Thx, @jpechane!

**Pull request merged**